### DISCUSSION TITLE
Jump to the error line by clicking on the line vscode (python, interactive window)

### DISCUSSION THREAD

**Discussion by zhaoguangyuan123** [microsoft/vscode-jupyter#10390] [Questions and Answers] [answered]
Same as the question in the link:
https://github.com/microsoft/vscode/issues/117783

I want to jump to the error Line by clicking the line on the output of the interactive window, but I find it is dark and I am not able to click. 

Is this caused by extension error?
if so, how can i solve the issue?

<img width="687" alt="image" src="https://user-images.githubusercontent.com/48136187/172900349-bb186f39-b141-4906-8b49-aca4a4a5d361.png">

**Comment by DonJayamanne** [comment]


**Reply by DonJayamanne** [reply]
please reply here https://github.com/microsoft/vscode-jupyter/issues/10409

**Comment by DonJayamanne** [comment, marked as answer]
Solution can be found here https://github.com/microsoft/vscode-jupyter/issues/10409
